docs/LitActions/intro.md

```diff
@@ -43,19 +43,30 @@ A proof is a particular application for a digital signature. For example, using
 
 Signing through Lit Actions opens up the possibilities of verifying information from external sources, such as from a Weather API. 
 
-## Permissionless Execution and Working With External Smart Contracts
+## Event Listening
 
-Right now, all Lit Actions have to be executed manually by a user. In order to enable the truly permissionless, "condition-based" execution of Lit Actions, we are working on developing our own native [event listening](https://litprotocol.notion.site/Event-based-triggers-for-PKPs-0b22532c571f4dffb08d6e84e9c5dd04) solution. 
-
-Currently, you could use something like [Tenderly Web3 Actions](https://docs.tenderly.co/web3-actions/intro-to-web3-actions) to listen for an on-chain event and then call a Lit Action as a result.
+Configure condition-based triggers for Lit Actions using the [event listener](/LitActions/singleExecution).
```

Comment:
I'd mark the event listener as a WIP - and put in a call for contributors to help build out additional features for this tool